src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -123,6 +121,24 @@
 						add("Check out the hydrate commands by typing \"::hydrate help\" in chat.");
 						add("A hydrated adventurer is an unstoppable adventurer.");
 						add("It's dangerous to go alone. Stay hydrated!");
+						add("Welcome Traveller. Nothing hurts morale like dehydration! Remember to drink water.");
+						add("People who don't believe in magic have obviously never had water!");
+						add("You're 70% water. Don't forget to stay hydrated!");
+						add("\"Thousands have lived without love, not one without water.\" - W.J. Auden");
+						add("Having trouble focusing? Sounds like you need to drink some water ;)");
+						add("\"If there is magic on this planet, it is contained in water.\" - Loren Eiseley");
+						add("Feeling low on energy? Drink some water!");
+						add("Nothing like RuneScape and a tall glass of ice cold water!");
+						add("I suppose we'll allow tea... For now.");
+						add("Type \"::hydrate next\" to set a reminder to stay hydrated!");
```

Comment:
The next command doesn't exactly set a reminder. Can you update this message to something like:
`Type \"::hydrate next\" to view the time remaining until the next hydration break!`